### ISSUE TITLE
Turning TV off with source level

### ISSUE THREAD

**Issue by rldn21** [GameDevHobby/Domoticz-LGTV-WebOS-Plugin#16]
Hello, how can i get the tv turned off with the source selector input via Domoticz, all the other levels work fine but when i have the "0" level configured with the "self.run("off")" command it does nothing:

In the plugin.py

if Unit == 3:   # TV source
                if Command == 'Set Level':
                    **if Level == 0:
                        self.run("off")**
                    if Level == 10:
                        #_tv.send_req_ircc("AAAAAQAAAAEAAAAAAw==") #TV Num1
                        self.run("app", "com.webos.app.livetv")
                        self.GetTVInfo()
                    if Level == 20:
                        #_tv.send_req_ircc("AAAAAgAAABoAAABaAw==") #Denon
                        self.run("app", "com.webos.app.hdmi2")
                        self.tvPlaying = "HDMI 2"
                    if Level == 30:
                        #_tv.send_req_ircc("AAAAAgAAABoAAABbAw==") #Netflix
                        self.run("app", "netflix")
                        self.tvPlaying = "Netflix"
                    if Level == 40:
                        #_tv.send_req_ircc("AAAAAgAAABoAAABcAw==") #PleX
                        self.run("app", "cdp-30")
                        self.tvPlaying = "PleX"
                    if Level == 50:
                        #_tv.send_req_ircc("AAAAAgAAABoAAABdAw==") #YouTube
                        self.run("app", "youtube.leanback.v4")
                        self.tvPlaying = "YouTube"
                    if Level == 60:
                        #_tv.send_req_ircc("AAAAAgAAABoAAAB8Aw==") #Elden Home
                        self.run("app", "com.itkey.plexclient")
                        self.tvPlaying = "XPlay"

                    self.tvSource = Level
                    self.SyncDevices()

Anyone?

**Comment by GameDevHobby** [comment]
Does it work when you turn it off with the "Status" device?

**Comment by rldn21** [comment]
> Does it work when you turn it off with the "Status" device?

Yes that works fine, but i want to turn the tv off with the selector level...

**Comment by GameDevHobby** [comment]
Have you tried adding "Off" as the last item in the list, as value 70? I'm not sure if 0 works as a value. "Off doesn't even show up in my selector, even though it's specified in the list.

**Comment by rldn21** [comment]
> Have you tried adding "Off" as the last item in the list, as value 70? I'm not sure if 0 works as a value. "Off doesn't even show up in my selector, even though it's specified in the list.

When debugging via Domoticz log, the selector sends a parameter "Off" when selecting Level 0/Off.
So i added a level 70 to the selector within Domoticz and within plugin.py added to the level selectors:
`                   
if Level == 70:
self.run("off")#_tv.turn_off()
self.tvPlaying = "Turned off TV"
`
When selecting the new Level 70 ("Off"), it sends correctly a parameter "Select Level" - 70 instead of parameter "Off" and it works perfectly like this (see Domoticz Log).
